packages/rainbowkit-wallets/src/lib/localStorageWallet.ts

```diff
@@ -0,0 +1,32 @@
+import { Wallet } from 'ethers'
+import { keccak256 } from 'ethers/lib/utils.js'
+
+export default class localStorageWallet {
+  static storageItemName = 'localstorage-wallet-seed'
+
+  public static async getWallet(): Promise<Wallet | undefined> {
+    try {
+      const value: string = localStorage.getItem(this.storageItemName) as string
+      if (!value) return undefined
+
+      return this.createWallet(value)
+    } catch (err) {
+      console.log(err)
```

Comment:
Better use console.error or console.warn for these cases. It's also a good idea to prefix it with a specific string, so we can easily locate it later (i.e. `console.error('failed to generate wallet:', err)`).